openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -68,6 +69,67 @@ OpenAssessment.BaseView.prototype = {
     },
 
     /**
+     * Checks the rendering status of the views.
+     * @returns {boolean} true if any step's view is still loading.
```

Comment:
messageView, I am still looking at.

leaderboardView we discussed.  I do not believe it needs an aria-live update.

I will take a look at staffAreaView, My initial thought was that it would not need an aria live region update.  That might be wrong though.